config/data.js

```diff
@@ -47,22 +45,49 @@ module.exports = async (deps) => {
     const currencies = ['USD', 'EUR'];
     const marketTypes = [0, 1];
 
-    await Promise.all(addresses.slice(1, 5).map(async (address) => {
+    const offerReceipts = await Promise.all(addresses.slice(1, 5).map(async (address) => {
       const addOffer = deps.contracts.MetadataStore.methods.addOffer(
-        tokens[Math.floor(Math.random()*tokens.length)],
+        tokens[1],
+        // TODO un hardcode token and check below
+        // tokens[Math.floor(Math.random() * tokens.length)],
```

Comment:
what is wrong with:
```
tokens[Math.floor(Math.random() * tokens.length)],
```
?

Reply:
Nothing wrong, it's just that creating a releasing escrows with tokens is way harder because we have to approve the SNT allowance. So I wanted to keep it simple.